spec/api/api_base_controller_spec.rb

```diff
@@ -0,0 +1,86 @@
+require "spec_helper"
+require "api/api_helper"
+
+include ApiHelper
+
+describe "Base API Controller" do
+
+  before do
+    allow(User).to receive(:is_archivist?).and_return(true)
```

Comment:
This fakes the `is_archivist?` call on `User` to always return `true` (so effectively, all users are archivists in these tests)